angularjs-portal-frame/src/main/webapp/portal/main/controllers.js

```diff
@@ -61,6 +62,19 @@ define(['angular'], function(angular) {
       that.user = result;
     });
   }]);
+  
+  app.controller('WelcomeController', ['$localStorage', '$sessionStorage','$scope', '$document', 'NAMES', '$modal', function($localStorage, $sessionStorage, $scope, $document, NAMES, $modal) {
+    $scope.openModal = function() {
+      if ($localStorage.hasSeenWelcome === false) {
+        $modal.open({
+          animation: $scope.animationsEnabled,
+          templateUrl: require.toUrl('../../my-app/layout/partials/welcome.html'),
```

Comment:
Isn't that just in the console though? I don't think anything user-facing breaks, right? I originally had it in frame, but then wouldn't every app using frame (student-parent-financial-portal, dnumber, etc) show the welcome popup for MyUW? I think that might be weird.